sql-mutations/insert-actor.sql

```diff
@@ -0,0 +1,4 @@
+insert into "actors" ("firstName", "lastName")
+values ('Johnny', 'Depp');
```

Comment:
You'll want `returning *` here, and in all your mutations. That's not critical now, but it will be when we do full-stack development.